Sources/Models/AvatarPosition.swift

```diff
@@ -73,16 +73,20 @@ public struct AvatarPosition {
     
     // The horizontal position
     public var horizontal: Horizontal
+
+    // The horizontal padding
+    public var padding: HorizontalEdgeInsets
```

Comment:
I'm not sure if this should be a property or an associated value of `Horizontal` 🤔

Reply:
I thought we should stick with the same pattern as LabelAlignment. Making it an associated value also makes it kind of a pain to read from. Are there any benefits in making it an associated value?

Reply:
Maybe call it horizontal padding then 👍

Reply:
👍🏻

Reply:
I feel like this property should still read `horizontalPadding`. What do you think? I know it reads `horizontalPadding.horizontal` then